src/analyze/analyze-security.c

```diff
@@ -143,12 +147,11 @@ static bool security_info_runs_privileged(const struct security_info *i)  {
         return isempty(i->user);
 }
 
-static int assess_bool(
-                const struct security_assessor *a,
-                const struct security_info *info,
-                const void *data,
-                uint64_t *ret_badness,
-                char **ret_description) {
+static int assess_bool(const struct security_assessor *a,
+                       const struct security_info *info,
+                       const void *data,
+                       uint64_t *ret_badness,
+                       char **ret_description) {
```

Comment:
Hmm. we use such indentation at many places. Please do not change them.

Reply:
hmm, no way around this. clang-format reindents every piece of code which is not explicitly left out with:
/* clang-format off */
... some custom formatting 
/* clang-format on */